docs/SG-NEx_m6Anet_tutorial.md

```diff
@@ -0,0 +1,151 @@
+
+# To download the readcount file
+wget https://sg-nex-data.s3.ap-southeast-1.amazonaws.com/data/processed_data/m6Anet/SGNex_A549_directRNA_replicate6_run1/data.readcount
+
+# Exit the A549 directory to return to our working directory
+cd ..
+```
+
+
+You may refer to this [page](https://github.com/GoekeLab/sg-nex-data/blob/update-docs-aws/docs/samples_with_RNAmod_data.tsv) for the list of URLs to access the processed data for m6Anet from different samples. 
+
+## **Running m6Anet**
+
+Now that all the required files are ready, we can proceed to run the m6Anet analysis. 
+
+
+```bash
+# Run m6Anet
+m6anet-run_inference --input_dir ./A549_directRNA_replicate6_run1 --out_dir ./A549_directRNA_replicate6_run1 --infer_mod_rate --n_processes 4
+```
+**Note: You can reduce the "n_processes" if you have lesser processes available in your machine. However, lowering the "n_processes" may increase the run time.**
+
+
+m6Anet returns a csv file `(data.result.csv.gz)` that contains the probability of modification as well as the modification stoichiometry for each candidate site. The first few lines of the file are shown below.
+
+
+<p align="center">
+    <img src="../images/A549_directRNA_replicate6_run1_result.png" alt="result" width="800"/>
+</p>
+
+Please refer to the [m6Anet documentation](https://m6anet.readthedocs.io/en/latest/quickstart.html) for more information regarding the output file. 
+<br>
+
+
+m6Anet also allows the pooling of replicates. It can be done by providing the file path to each replicate's folder containing the json, index and readcount files. 
+
+Example:
+```
+m6anet-run_inference --input_dir ./replicate1 ./replicate2 ./replicate3 --out_dir ./result --infer_mod_rate --n_processes 4
+```
+<br>
+
+#### **NOTE: This is a short tutorial to demonstrate the usage of m6Anet on the SG-NEx data. Please refer to the [m6Anet documentation](https://m6anet.readthedocs.io/en/latest/quickstart.html) for the complete workflow to identify m6A modification.** 
+<br>
+
+### **Access to the required files for complete workflow** 
+
+If you wish to run the complete workflow of m6Anet, you can access all the required files (fast5, fastq and bam files) from the [SG-NEx S3 bucket](https://github.com/GoekeLab/sg-nex-data/blob/update-docs-aws/docs/samples.tsv). 
+
+
+```bash
+# Note: Please make sure to replace the "sample_alias" with your sample name
+
+# To download the fast5 file
+aws s3 sync --no-sign-request s3://sg-nex-data/data/sequencing_data_ont/fast5/<sample_alias> ./
+
+# To download the fastq file
+aws s3 sync --no-sign-request s3://sg-nex-data/data/sequencing_data_ont/fastq/<sample_alias> ./
+
+# To download the bam file
+aws s3 sync --no-sign-request s3://sg-nex-data/data/sequencing_data_ont/bam/transcriptome/<sample_alias> ./
+```
+
+Meanwhile, you can also access the transcriptome fasta files and the gtf file (annotation file) using the following command.
```

Comment:
```suggestion
Meanwhile, you can also access the transcriptome fasta file and the gtf file (annotation file) using the following command.
```

Reply:
I think there are more than one fasta file ? One is the fasta itself and the other one is the index file

Reply:
oh yes, that's correct, I think you can ignore this one then.